sonar-project.properties

```diff
@@ -5,6 +5,6 @@ sonar.organization=ag-grid
 sonar.projectKey=ag-charts-community-latest
 sonar.projectBaseDir=./
 sonar.host.url=https://sonarcloud.io
-sonar.exclusions=**/tsconfig.spec.json,**/LICENSE.html,**/*.md,**/*.mdoc,**/dist,**/node_modules,**/typings,**/lib,**/*spec.js,packages/*/tools/**,packages/*/src/**/test/**,**/*.test.*,**/__mocks__/**
+sonar.exclusions=**/tsconfig.spec.json,**/LICENSE.html,**/*.md,**/*.mdoc,**/dist,**/node_modules,**/typings,**/lib,**/*spec.js,packages/*/tools/**,packages/*/src/**/test/**,**/*.test.*,**/__mocks__/**,packages/ag-charts-enterprise/src/license/md5.ts
```

Comment:
This `md5.ts` file throws a lot of errors like this - https://sonarcloud.io/project/issues?impactSeverities=MEDIUM&rules=typescript%3AS2234&issueStatuses=OPEN%2CCONFIRMED&id=ag-charts-community-latest&open=AYsEn8c2Mhz_bILO5yA8 - which are false issues in this instance and I didn't want to mess with the algorithm.